notice_and_comment/templates/regulations/main-header.html

```diff
@@ -0,0 +1,20 @@
+<div class="main-head">
```

Comment:
Mind adding a `{% comment %}` to explain why this override is needed? I think for the "Partnership between" text?